Frontend/src/components/Table/Cell.svelte

```diff
@@ -0,0 +1,70 @@
+<script>
+  export let backgroundColor = "white";
+  export let isImage = false;
+  export let value;
+  export let rowHeight;
+
+  let fontColor = "black";
+
+  function brightnessByColor(color) {
+    var color = "" + color,
+      isHEX = color.indexOf("#") == 0,
+      isRGB = color.indexOf("rgb") == 0;
+    if (isHEX) {
+      var m = color.substr(1).match(color.length == 7 ? /(\S{2})/g : /(\S{1})/g);
+      if (m)
+        var r = parseInt(m[0], 16),
+          g = parseInt(m[1], 16),
+          b = parseInt(m[2], 16);
+    }
+    if (isRGB) {
+      var m = color.match(/(\d+){3}/g);
+      if (m)
+        var r = m[0],
+          g = m[1],
+          b = m[2];
+    }
+    if (typeof r != "undefined") return (r * 299 + g * 587 + b * 114) / 1000;
+  }
+
+  $: if (brightnessByColor(backgroundColor) < 125) {
+    fontColor = "white"; // adaptive font color for darker highlight
+  }
+</script>
+
+<td style={`background-color: ${backgroundColor}; color: ${fontColor}; height: ${rowHeight}px;`}>
```

Comment:
why did you decide not to have a generalized `style` function as before? Could there not be more style changes that we would want to make in the future?

Reply:
I think in order to keep the code clean it's better to implement a more generalized style function when we need it.